ePrint IACR.js

```diff
@@ -108,7 +107,13 @@ function doWeb(doc, url) {
 			ZU.processDocuments(articles, scrape);
 		});
 	} else {
-		scrape(doc, url)
+		if (url.search(/\.pdf$/)!= -1) {
+			//go to the landing page to scrape
+			//we use http to prevent (somewhat mysterious) same-origin violations leading to permission denied errors
+			url = url.replace(/\.pdf$/, "").replace(/^https/, "http")
```

Comment:
@zuphilip @adomasven -- testing this on 4.0.29 in Firefox -- I'm a little confused as to why I need to switch to http, but it's the only way I was able to make that work